src/app/faqs/faq-list/faq-list.component.html

```diff
@@ -0,0 +1,154 @@
+<div class="d-flex row-hl">
+  <!-- Category Selector -->
+  <div class="p-2 w-20">
+    <div *ngFor="let faqCategory of faqCategories">
+      <button
+        mat-raised-button
+        [color]="filterFaqCategory?.id == faqCategory.id ? 'accent' : ''"
+        class="card mb-3 w-100"
+        (click)="onClickCategory(faqCategory)"
+      >
+        <div
+          class="d-flex align-items-center justify-content-between flex-row p-3"
+        >
+          <i class="fa mr-4" [ngClass]="faqCategory.icon"></i>
+          <div class="d-flex flex-column align-items-center w-100">
+            <mat-card-title>
+              {{ faqCategory.title }}
+            </mat-card-title>
+            <span class="font-weight-normal line-height-normal text-wrap">
+              {{ faqCategory.description }}
+            </span>
+          </div>
+        </div>
+      </button>
+    </div>
+    <button
+      *ngIf="userIsAdmin"
+      mat-raised-button
+      color="primary"
+      class="w-100"
+      (click)="onClickAddCategory()"
+    >
+      Add Category
```

Comment:
kill this - as part of the FAB + icon have it be either create new faq or new category